posts/2022-01-14-construct-vendia-univseral-application-using-domain-driven-design.md

```diff
@@ -0,0 +1,177 @@
+
+
+## Uni Development Lifecycle
+
+Uni is a cross-cloud, cross-party, cross-account, cross-region data and code sharing application powered by Vendia's serverless distributed ledger technology. A uni is composed of [nodes](https://www.vendia.net/docs/share/terms-and-definitions#node) modeling [participants](https://www.vendia.net/docs/share/terms-and-definitions#participants), each of which possesses an ordered, replicated data model in the form of a distributed [ledger](https://www.vendia.net/docs/share/terms-and-definitions#ledger).
+
+A Uni software development life cycle has five stages: Model, Design, Deploy, Operate, and Scale. Since a Uni is serverless, deployment, operations, and scaling are easy. Once you have the schema design, deployment is just a couple of clicks, and operations are handled by Vendia.  Vendia is responsible for scalability and performance. This leaves us with data modeling and design. Data models are the core of any software. In Uni, the design phase involves constructing a schema that is a deployable artifact and leads to the creation of Uni. In this blog, we will focus on data modeling and design.
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/96793170/151029339-3c97b4d4-e66a-4b37-ae21-6200e22cb042.png" />
+</p>
+<p align="center">Figure 3 - Uni Development Lifecycle Stages</p>
+
+
+## Uni Schema Anatomy
+
+Let’s imagine a simple Uni schema that provides a datastore for core customer data attributes. Figure 4 breaks the Uni schema into parts and tries to map it to concepts you use in a relational database. This will help you quickly get started on building your own Uni. 
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/96793170/151029529-fa688c30-766c-4803-bc59-3b953e3ae0c4.png" />
+</p>
+<p align="center">Figure 4 - Example Customer Uni</p>
+
+
+## Applying DDD to Uni 
+
+
+### Pattern 1: Single Domain, Single Bounded Context, and a Single Uni
+
+This pattern is well suited for organizations who want to build small custom applications surrounding complex enterprise applications, or organizations who want to construct a new application to prove a new business idea. 
+
+In this pattern, a Uni model contains a data model for a single bounded context. For example, the CRM team models customer account details in Bounded Context 1. 
+
+
+<p align="center">
+  <img src="https://user-images.githubusercontent.com/96793170/151029629-68861fc8-daaf-4dc5-ab21-5269df0a5068.png" />
+</p>
+<p align="center">Figure 5 - Pattern 1</p>
+
+**You can use the below resources to practice and deploy pattern 1.**
+
+<details>
+<summary>Schema</summary>
+[schema.json](https://github.com/vendia/examples/blob/7792069dd40c6822279305b201965685aaac5a08/share/data-modeling/domain-driven-design/pattern1/schema/schema.json)
+</details>
+<details>
+<summary>Details and deployment instructions</summary>
```

Comment:
Also, I think you need to have a newline after `<summary>Details and deployment instructions</summary>` for the formatting to work as expected. There has to be a blank line between.